crates/biome_js_analyze/src/lint/correctness/use_jsx_key_in_iterable.rs

```diff
@@ -36,6 +38,16 @@ declare_lint_rule! {
     /// data.map((x) => <Hello key={x.id}>{x}</Hello>);
     /// ```
     ///
+    /// ## Options
+    ///
```

Comment:
Can you please **explain** the option to our users? Also, you can add an example that **uses** these options: https://github.com/biomejs/biome/blob/main/crates/biome_analyze/CONTRIBUTING.md#using-rule-options